### ISSUE TITLE
Notable on Linux creates and renames file but does not save content

### ISSUE THREAD

**Issue by tracyde** [notable/notable#921]
Notable v1.8.1 on Linux (Snap, AppImage, and Deb) creates and renames the underlying markdown file and `title`, `created`, `modified` tags but does not actually write the body of the file. `title` also stays 'Untitled'.

Tried this on Chrome OS crostini (deb stretch and buster) along with KDE Neon, same behavior on both.

No errors being thrown that I can see.

**Comment by fabiospampinato** [comment]
Another report of potentially the same issue: https://www.reddit.com/r/Notable/comments/e051x3/possible_to_turn_on_verbosity_or_logging/

**Comment by vsimkus** [comment]
Same issue on Debian Bullseye installed via snap.

Would love to use Notable for tracking my research notes and tasks.

**Comment by fabiospampinato** [comment]
I tried reproducing this, but:

- Crostini seems too complicated to set-up a VM for.
- KDE Neon refused to boot under Parallels.
- Gnome Bullseye wasn't able to find `snapd` for some reason, and the `deb` had a missing dependency so couldn't be installed.

I'll take another look at this in the future, in the meantime if anybody has any more info on why this isn't working or on how to reproduce this I'm all ears.

**Comment by ekorudiawan** [comment]
Same here, with Ubuntu 18.04 installed from snap

**Comment by fabiospampinato** [comment]
@ekorudiawan I think the snap issue is a different one, the used sandbox there is not appropriate for what we need, can you try installing the deb instead? https://github.com/notable/notable/releases/download/v1.8.4/notable_1.8.4_amd64.deb

**Comment by antonpiatek-imanage** [comment]
I installed this because it is packaged in the snap store and will get updates, if there is a suggestion that the wrong sandbox is in use, is there a test snap I can try as an alternative? I'm on `Ubuntu 19.10 eoan` with snap `stable:    1.8.4 2020-02-21`

**Comment by fabiospampinato** [comment]
@antonpiatek-imanage I haven't published one yet, as I'm waiting for a manual approval for enabling the "classic" confinement in the store, but this test snap here has the classic confinement enabled: https://drive.google.com/file/d/1leLCIgUHaVlOtZseAwABFkRvyPY8leFz/view?usp=sharing

I'm not aware of any significant improvements this would have other the regular [deb](https://github.com/notable/notable/releases/download/v1.8.4/notable_1.8.4_amd64.deb) however.

**Comment by antonpiatek-imanage** [comment]
@fabiospampinato Thanks, the main improvement is that any new versions in the snap store will upgrade on my laptop automatically, whereas a custom deb will never auto-update

**Comment by antonpiatek-imanage** [comment]
FWIW I can confirm the linked deb does work correctly

**Comment by fabiospampinato** [comment]
@antonpiatek-imanage Yeah hopefully the issue with the store will be resolved soon, I'm still waiting for them to manually enable me to ship the app with classic confinement.

**Comment by aligoren** [comment]
> @ekorudiawan I think the snap issue is a different one, the used sandbox there is not appropriate for what we need, can you try installing the deb instead? https://github.com/notable/notable/releases/download/v1.8.4/notable_1.8.4_amd64.deb

It worked on Ubuntu Linux. Thanks. You saved me.

**Comment by fabiospampinato** [comment]
I've just fixed the issue with the snap.

Whatever other issues are left, as mentioned by @tracyde, I think are Crostini-specific issues as I can't reproduce them in a regular Ubuntu installation. Maybe the snap now works there too?
